### PR TITLE
Add fork steps

### PR DESCRIPTION
**Pull request by pamella**
## Description

- Add fork steps
- Fix broken slides link

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt (refactoring existing code)
- [X] Another type of change, e.g. documentation

## Checklist:

- [x] My change requires frontend review.
- [X] My change requires documentation updates.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires dependencies updates.
- [x] I have updated the dependencies accordingly.
- [ ] My change affected critical flows & code.
- [ ] I have tested the critical flows & code accordingly.
- [ ] My change requires environment variables updates.
- [x] I have described the environment variables accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,89 @@
 # :star: pj-git
 Repositório auxiliar ao treinamento de Git para a PJ. (Dez, 2019)
 
+
```
Comment (larissarsilva) [inline README.md:4]: teste

File: README.md [hunk 2]
```diff
@@ -1,9 +1,36 @@
 # :star: pj-git
 Repositório auxiliar ao treinamento de Git para a PJ. (Dez, 2019)
```
Comment (sofiawolf2) [inline README.md:3]: testando

File: README.md [hunk 3]
```diff
@@ -1,8 +1,89 @@
...
```
Comment (Gabriellopess) [inline README.md:3]: teste..

File: README.md (hunk 1 above)
Comment (pamella) [reply README.md]: Corrigido!

Comment (pamella) [reply README.md:4]: @larissarsilva  Corrigido!

**Pull request merged**